hack/manifests/base/kustomization.yaml

```diff
@@ -0,0 +1,3 @@
+resources:
```

Comment:
Do you think it makes sense to have more manifests? We have local profile but we need integration and production as well.

Reply:
At the moment I added the testing folder, and from there we can evolve in case that it's needed ;-)

Reply:
OK, make sense